CHANGELOG

```diff
@@ -1,4 +1,6 @@
 0.39.15
+ - fix: fix accessing inherited non-scalar temporary or plugin features
+   for RTDC-Hierarchy class (#165)
```

Comment:
Please remove the dash in `RTDC-Hierarchy` (it's not the class name)

Reply:
I think I added an underscore for RTDC_DCOR and RTDC_HDF5 because it would be unreadabe otherwise. Then I added the underscore for RTDC_Hierarchy to be consistent. I think it's not worth changing. Yes, please add the underscore.